src/doc/building_visit/BuildingOnWindows/ConfiguringWithCMakeGUI.rst

```diff
@@ -147,49 +140,48 @@ individual entries (not groups) will generate a brief description.
 
 Click the **Configure** button again to have CMake check the prerequisites for 
-building parallel VisIt. If the prerequisites are met then some new cache 
-entries related to MPI will be created.  If not, the MPI entries may have to
-be modified by hand.
+building parallel VisIt_. 
+If the prerequisites are met then some new cache entries related to MPI will be created.  
+If not, the MPI entries may have to be modified by hand.
 
 Suppressing Regeneration
 ~~~~~~~~~~~~~~~~~~~~~~~~
 The solution file that CMake creates has a project called *ZERO_CHECK* that is 
-occasionally invoked to regenerate the projects. This can be highly undesirable 
-during development, since it may be triggered  during a build and can cause 
-numerous projects to be reloaded into the VS IDE, wasting time unnecessarily. 
-To avoid this behavior, you can create a new CMake cache entry named 
-*CMAKE_SUPRESS_REGENERATION*,  with type *BOOL* and make sure that it is 
-checked. If you made this change click **Configure** again.
+occasionally invoked to regenerate the projects. 
+This can be highly undesirable during development, since it may be triggered during a build and can cause numerous projects to be reloaded into the VS IDE, wasting time unnecessarily. 
+To avoid this behavior, you can create a new CMake cache entry named *CMAKE_SUPRESS_REGENERATION*,  with type *BOOL* and make sure that it is checked. 
+If you made this change click **Configure** again.
 
 You can automate this step in your host.cmake file by adding this line to your host.cmake file:
  ``set(CMAKE_SUPPRESS_REGENERATION TRUE)``
 
-Note that setting this flag means that CMake won't automatically reconfigure 
-from within the VS IDE when changes are made to the build scripts 
-(CMakeLists.txt) or Cache entries.  You will have to manually reconfigure.
-Once reconfigured, Visual Studio will notify you the project files have been
-modified and prompt you to reload.
+Note that setting this flag means that CMake won't automatically reconfigure from within the VS IDE when changes are made to the build scripts (CMakeLists.txt) or Cache entries.  
+You will have to manually reconfigure.
+Once reconfigured, Visual Studio will notify you the project files have been modified and prompt you to reload.
 
 Generate
 ~~~~~~~~
 The *Generate* step creates the Visual Studio project and solution files. 
-Make sure any changes made to the cache entries have been *Configured* and 
-that no entries remain red, then click the **Generate** button.
+Make sure any changes made to the cache entries have been *Configured* and that no entries remain red, then click the **Generate** button.
 
 Compile
 ~~~~~~~
-Open the generated VisIt.sln file with Visual Studio (it may take awhile to
-load all the project file).  Select desired Configuration and Build solution.
+Open the generated *VisIt.sln* file with Visual Studio (it may take awhile to
+load all the project files). 
+Select desired Configuration and Build solution.
```

Comment:
Change "Select desired" to "Select the desired".

Reply:
done